### PR TITLE
add editorial guide; fix layouts

### PR DESCRIPTION
**Pull request by juliaferraioli**
Layout changes were necessary to maintain font consistency throughout the pages.

Primary review requested on `editorial-guide.md`, please!

### CODE REVIEW

File: content/contribute/editorial-guide.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+As the editor for a new story, you will be given access to a folder containing the following resources:
+
+* a transcript in the desired output template
+* the audio recording
+* subtitle files
+
+All of your tasks will be completed in the transcript document and tracked in the GitHub issue.
+
+Editing a story can be broken down into the following steps:
```
Comment (amcasari) [inline content/contribute/editorial-guide.md:20]: How do you feel about making this an ordered list?

File: content/contribute/editorial-guide.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+1. add punctuation where needed
+1. tighten up language and clarify ambiguities
+
+### Break into paragraphs
+
+While the transcript will already have some paragraphs based on speaker and timing, if there are additional breaks that you find would be helpful, please add them.
+
+### Fine-tune title
+
+The transcript will have a draft title, but please feel free to fine-tune it if you have a more appropriate one.
```
Comment (amcasari) [inline content/contribute/editorial-guide.md:45]: Also recommend adding in "Puns are greatly appreciated and highly encouraged."

File: content/contribute/editorial-guide.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+If there are references to people, technologies, or events that you think could benefit from a reference, please add them as links in the transcript.
+
+### Suggest tags (optional)
+
+Tags help people find other interesting stories! Add the tags you think would be most helpful for others.
+
+See [opensourcestories.org/tags](https://www.opensourcestories.org/tags/) for existing tags
+
+### Write social media copy (optional)
```
Comment (amcasari) [inline content/contribute/editorial-guide.md:65]: Bookmark for when we create a "social-guidelines" doc -

Add in section on images, copyright, best practices for alt-text.

File: content/contribute/editorial-guide.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+Social media, that fickle medium. If you are so inclined, please write a version (or two, or three) of text that we can use to promote the story on Mastodon, Twitter, LinkedIn, etc…
```
Comment (amcasari) [inline content/contribute/editorial-guide.md:67]: 😂

File: content/contribute/editorial-guide.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+* Twitter limited to 280 chars unless subscriber to Twitter Blue
+* LinkedIn allows up to 5075 chars for message body
+* Mastodon defaults to 500 char limit in post; may vary by server
+
+## Tracking progress
+
+Please keep track of your progress in editing in the issue for the story. Once you have finished your tasks, assign the issue to the person who is named as the publisher for the story.
```
Comment (amcasari) [inline content/contribute/editorial-guide.md:75]: Recommend adding in after "for the story.":

"We would say no update is too small, but maybe "Opened a file 🤘" is. 

In general, letting us know where you're flowing, where you get stuck, and how we can help benefits everyone."

File: content/contribute/editorial-guide.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+We are all human! If there are delays of more than ~4 days, or you can no longer be the editor, just let us know on the issue.
```
Comment (amcasari) [inline content/contribute/editorial-guide.md:77]: <3

**Review by amcasari — COMMENTED** [review]
Overall - fantastic and well structured. Thank you <3

I had some questions about some points that were not completely clear for me. Other areas where I get more verbose, but not necessary.

File: content/contribute/editorial-guide.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
+---
+title: "Editorial Guide"
+draft: true
+summary: "This document describes the responsibilities for the person or people assigned to the editor role for an open source story, as outlined in the editing section of the `new-story` issue template."
+---
+This document describes the responsibilities for the person or people assigned to the editor role for an open source story, as outlined in the editing section of the `new-story` issue template.
+
+> Would you like to be an editor for Open Source Stories? Let us know by sending us an email at [contribute@opensourcestories.org](mailto:contribute@opensourcestories.org?subject=I%27d%20like%20to%20volunteer%20as%20an%20editor!)!
```
[the remark this answers is not in the document]
Comment (juliaferraioli) [reply content/contribute/editorial-guide.md:8]: that's a pretty outdated model. It doesn't actually cut down on spam anymore -- bots even can extract email addresses from images these days.

File: content/contribute/editorial-guide.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+## Editing process
+
+As the editor for a new story, you will be given access to a folder containing the following resources:
+
+* a transcript in the desired output template
```
[the remark this answers is not in the document]
Comment (juliaferraioli) [reply content/contribute/editorial-guide.md]: The transcript is from the transcription service, already in the template we want (has placeholders for the fields). You can see the template in our template library in Google Docs.

Added the file types. The editor doesn't name or add any files, so I don't think a naming convention is relevant?

File: content/contribute/editorial-guide.md (hunk 3 above)
Comment (juliaferraioli) [reply content/contribute/editorial-guide.md:65]: Converted to issue!

File: content/contribute/editorial-guide.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+* the audio recording
```
[the remark this answers is not in the document]
Comment (juliaferraioli) [reply content/contribute/editorial-guide.md]: Added file type and addressed the naming convention in earlier comment.

File: content/contribute/editorial-guide.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+* subtitle files
```
[the remark this answers is not in the document]
Comment (juliaferraioli) [reply content/contribute/editorial-guide.md]: Added file type and addressed the naming convention in earlier comment.

**Pull request merged**